test/integration/MorphoVaultV1IntegrationDonationTest.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright (c) 2025 Morpho Association
+pragma solidity ^0.8.0;
+
+import "./MorphoVaultV1IntegrationTest.sol";
+
+contract MorphoVaultV1IntegrationDonationTest is MorphoVaultV1IntegrationTest {
+    using MorphoBalancesLib for IMorpho;
+
+    address internal immutable donor = makeAddr("donor");
+
+    function testSharesDonationResistance(uint256 initialAssets, uint256 donatedAssets, uint256 elapsed) public {
```

Comment:
otherwise it could be misleading
```suggestion
    function testSharesDonationResistanceIfNoAllocation(uint256 initialAssets, uint256 donatedAssets, uint256 elapsed) public {
```